config/form_profile_mappings/21-686C.yml

```diff
@@ -0,0 +1,5 @@
+claimantEmail: [contact_information, email]
```

Comment:
This is tricky, we don't actually show these fields unless you mark that you're not a Veteran. Which means this info will probably be erased before we get to it.

I'm not actually sure how to handle this, since someone could be filling this out as a Veteran or as someone related to a Veteran.

Reply:
how about we prefill both claimant and veteran fields?

Reply:
That's fine.